crates/bitcoind_rpc/src/lib.rs

```diff
@@ -0,0 +1,243 @@
+//! This crate is used for emitting blockchain data from the `bitcoind` RPC interface (excluding the
```

Comment:
" from the `bitcoind` RPC interface (excluding the RPC wallet API)."

What do you mean? Is this saying that we don't use any wallet-related RPCs?

Reply:
Yes, how do you recommend it worded?

Reply:
This crate is used for emitting blockchain data from the `bitcoind` RPC interface. It doesn't use the wallet RPC API, so that you can use this crate even with wallet-disabled Bitcoin Core nodes.

How does this sound?